StardewArchipelago/GameModifications/CodeInjections/EntranceInjections.cs

```diff
@@ -51,6 +51,14 @@ public static void Initialize(IMonitor monitor, ArchipelagoClient archipelago, E
             catch (Exception ex)
             {
                 _monitor.Log($"Failed in {nameof(PerformWarpFarmer_EntranceRandomization_Prefix)}:\n{ex}", LogLevel.Error);
+                var targetPosition = new Point(tileX, tileY);
+                if (_entranceManager.TryGetEntranceReplacement(Game1.currentLocation.Name, locationRequest.Name, targetPosition, out var replacedWarp))

```

Comment:
I think a different try{}catch(){} could be placed somewhere smart in EntranceManager (Or GameLocationExtensions?) That could both provide more information about exactly what went wrong, and avoid calling the problematic code a 2nd time and risking a full on crash